packages/interactivity/src/directives.js

```diff
@@ -241,6 +241,13 @@ export default () => {
 			const contextStack = useMemo( () => {
 				if ( defaultEntry ) {
 					const { namespace, value } = defaultEntry;
+					// Check that the value is a JSON object. Send a console warning if not.
+					if ( SCRIPT_DEBUG && ! isPlainObject( value ) ) {
```

Comment:
When this is published as a package, we don't want to require these arbitrary global variables to be declared, so we should add a `typeof` check like is done in the warning package (this may require formatting):

```suggestion
					if ( typeof SCRIPT_DEBUG !== 'undefined' && SCRIPT_DEBUG === true && ! isPlainObject( value ) ) {
```

https://github.com/WordPress/gutenberg/blob/649eddf5fc9b19ebae0426cb99b4d3894c4e5a11/packages/warning/src/index.js#L6-L8

Otherwise, when this is used in other environments we'd get a ReferenceError.

I'm a little bit concerned that we don't lint this type of thing 😕